js/gluon.js

```diff
@@ -95,7 +95,7 @@ $(function() {
     });
     
     // Scroll to anchor from navbar-nav links
-    $(document).on('click', ".nav-link", function(event) {
+    $(document).on('click', 'a[class*="nav-link"][data-toggle="collapse"]', function(event) {
```

Comment:
Wouldn't `[data-target=".navbar-collapse.show"]` be even a better selector, then `data-toggle`?

Reply:
`collapse` is intended only for a collapsable item of a menu, and only the floating nav-bar items have it (see index.html). Actually these are the only ones that have `data-toggle`, and also the only ones that have `data-target`. That's why I don't see any difference.

Reply:
Ok, I was only wondering if it's future proof enough.